_includes/angular/controlleras.html

```diff
@@ -0,0 +1,9 @@
+<!-- avoid -->
+<div ng-controller="Customer">
+{% raw %}    {{ name }}{% endraw %}
+</div>
+
+<!-- recommended -->
+<div ng-controller="Customer as customer">
```

Comment:
The only thing with examples like this is, is it does get confusing for those with not a lot of Angularjs experience. For example someone was putting the <div controller=""> syntax in his HTML in places they didn't need because examples had that syntax.